web/src/stream_pill.ts

```diff
@@ -67,33 +77,43 @@ export function get_user_ids(pill_widget: StreamPillWidget | CombinedPillContain
 
 export function append_stream(
     stream: StreamSubscription,
-    pill_widget: CombinedPillContainer,
+    pill_widget: StreamPillWidget | CombinedPillContainer,
+    show_subscriber_count = true,
```

Comment:
It would be better to do the refactors to add options like this in their own commits, for future reference, since that helps a reviewer model how much risk there is of breaking other pill typeahead components.